corehq/apps/export/tasks.py

```diff
@@ -107,20 +112,44 @@ def rebuild_saved_export(export_instance_id, last_access_cutoff=None, manual=Fal
     if manual:
         if status.not_started() or status.missing():
             # cancel pending task before kicking off a new one
+            # (May result in many revoked tasks if users kick off the
+            # same export several times.)
             download_data.task.revoke()
         if status.started():
             return  # noop - make the user wait before starting a new one
     else:
         if status.not_started() or status.started():
             return  # noop - one's already on the way
 
+    queue = EXPORT_DOWNLOAD_QUEUE if manual else SAVED_EXPORTS_QUEUE
+    queue_len = get_queue_length(queue)
+    if queue_len == QUEUE_LENGTH_LOGGING_THRESHHOLD:
+        # This makes the assumption, which may not be true, that the
+        logger.info('Queue {} at 100 tasks. Logging task data.'.format(queue))
+        for task in get_queue_tasks(queue):
+            inst_str = '; '.join((
+                'ID: {}'.format(task['export_instance']['id']),
+                'Name: {}'.format(task['export_instance']['name']),
+                'Domain: {}'.format(task['export_instance']['domain']),
+            ))
+            task_str = '; '.join((
+                'Task ID: {}'.format(task['id']),
+                'State: {}'.format(task['state']),
+                'Status: {}'.format(task['status'].state),
+                'Start time: {}'.format(task['start_time'] or 'N/A'),
+                'Progress: {}%'.format(task['status'].progress.percent),
+                'Error: "{}"'.format(task['status'].progress.error_message or "N/A"),
+                'Export instance: ({})'.format(inst_str),
+            ))
+            logger.info(task_str)
```

Comment:
Same here, could you stick this in a helper function?  Looks like these two log calls are very similar, so it may make sense to abstract them for that reason anyways.